kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/index.vue

```diff
@@ -40,29 +42,50 @@
       <div class="bottom-nav-container">
         <KButtonGroup>
           <KRouterLink
-            v-if="selectedResources.length > 0"
+            v-if="
+              selectedResources.length > 0 &&
+                $route.name !== PageNames.LESSON_PREVIEW_SELECTED_RESOURCES
+            "
             :to="{ name: PageNames.LESSON_PREVIEW_SELECTED_RESOURCES }"
           >
             {{ selectedResourcesMessage }}
           </KRouterLink>
           <KButton
             primary
+            :disabled="isSaving"
             :text="saveAndFinishAction$()"
-            @click="closeSidePanel"
+            @click="save"
           />
         </KButtonGroup>
       </div>
     </template>
+
+    <KModal
+      v-if="isCloseConfirmationOpen"
```

Comment:
This is absolutely a nitpick but somehow the "close" and "open" combined is confusing, although I do know what it means, and it makes sense to label it this way once I think about it. 😹  let me see if I might have a recommendation for another phrasing, but definitely not blocking or anything you have to change.

Reply:
hehe okay thank you